hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/KeyValueStreamDataChannel.java

```diff
@@ -39,4 +153,125 @@
   ContainerProtos.Type getType() {
     return ContainerProtos.Type.StreamWrite;
   }
+
+  @Override
+  public int write(ReferenceCountedObject<ByteBuffer> referenceCounted)
+      throws IOException {
```

Comment:
Hi @szetszwo. I just tested this PR, but found a lot of AbstractMethodError. [Here is datanode log](https://issues.apache.org/jira/secure/attachment/13041652/ozone-root-datanode-9-29-173-57-1.log). Can you confirm that?
<img width="1411" alt="image" src="https://user-images.githubusercontent.com/13825159/160279681-08fb013e-b93c-45b4-a161-7ebccf6d3781.png">

Reply:
@captainzmc , I guess you somehow were using some old version of Ratis/Ozone since you got AbstractMethodError. Please try deleting the maven cache and then rebuild everything.

This change should work since it already has passed all the checks in https://github.com/apache/ozone/runs/5687893931

Reply:
Oops, my mistake, I didn't replace all the jars. Let me recompile and retest.

Reply:
I recompiled the code and the tests were fine.